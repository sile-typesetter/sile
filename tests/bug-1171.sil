\begin[class=jplain,layout=tate]{document}
\nofolios
\noindent
\set[parameter=font.family,value=Noto Serif CJK JP,makedefault=true]
\define[command=emoji]{\font[family=Twemoji Mozilla]{\process}}
\emoji{ðŸ˜‰} \emoji{ðŸ¤£}
\end{document}
