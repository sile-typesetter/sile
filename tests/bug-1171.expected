Set paper size 	595.275597	841.8897729
Begin page
Push color	1.0000	0.8000	0.3020
Mx 	542.4079
My 	97.6592
Set font 	Twemoji Mozilla;10;400;;normal;;TTB
T	7876 a=10.0000	()
Pop color
Push color	0.4000	0.2706	0.0000
Mx 	10.0000
T	7893 a=10.0000 7894 a=10.0000	(ðŸ˜‰)
Pop color
Push color	1.0000	0.7961	0.2980
Mx 	542.4079
My 	110.8167
T	9105 a=10.0000	()
Pop color
Push color	0.3961	0.2784	0.1059
T	9106 a=10.0000	()
Pop color
Push color	1.0000	1.0000	1.0000
T	9107 a=10.0000	()
Pop color
Push color	0.3961	0.2784	0.1059
T	9108 a=10.0000	()
Pop color
Push color	0.3922	0.6667	0.8667
T	9109 w=10.0000	(ðŸ¤£)
Pop color
End page
Finish
